### PR TITLE
Supporting same day deliveries

### PR DESCRIPTION
**Pull request by gedzubo**

### CODE REVIEW

File: lib/mister_pasha_api/operations/fetch_time_slots.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        {
+          available_time_slots: available_time_slots(body["liste_creneau"]),
+        }
+      end
+
+      def available_time_slots response
+        result = []
+        response.each do |delivery_date, slots|
+          slots.each do |slot|
+            result << TimeSlot.new(slot, delivery_date)
```
Comment (newbray) [inline lib/mister_pasha_api/operations/fetch_time_slots.rb]: is this delivery_date or delivery_time?

Comment (gedzubo) [reply lib/mister_pasha_api/operations/fetch_time_slots.rb]: date as "2019-06-03".

Comment (newbray) [reply lib/mister_pasha_api/operations/fetch_time_slots.rb]: the variable in TimeSlot is called delivery_time...

Comment (gedzubo) [reply lib/mister_pasha_api/operations/fetch_time_slots.rb]: Got spot 😱 will fix.

Comment (newbray) [reply lib/mister_pasha_api/operations/fetch_time_slots.rb]: 😱

Comment (newbray) [reply lib/mister_pasha_api/operations/fetch_time_slots.rb]: 👍

**Pull request merged**